content/stranger_case/report.md

```diff
@@ -0,0 +1,175 @@
+  
+#### Raphael Mabar {#rm}
+
+{{< imgWithClass "/images/stranger_case/tosalios.png" "Raphael Mabar" "tosalios.png" "" "stranger-case-pp" >}}
+
+- **Twitter**: [https://twitter.com/Tosalios](https://twitter.com/Tosalios)
+- **Facebook**: [https://www.facebook.com/profile.php?id=100091989057511](https://www.facebook.com/profile.php?id=100091989057511)
+- **Alias**: Tosalios
+- **Blog**: [https://tosalio.blog/](https://tosalio.blog/)
+
+His Blog has 3 snapshots on the wayback machine but nothing seems relevant except it runs WordPress, and it is associated with the username: lenainkiller0119.
+On his Twitter page we found several pieces of information: the first is that he is a fan of cycling and more particularly of the rider Thibaut PINOT. We also know that he follows the forum [https://www.jeuxvideo.com/](https://www.jeuxvideo.com/) maybe we can find something interesting there. The last big piece of information we learn is that he's a fan of the game Clash of Clans. 
+
+![Tosalios](/images/stranger_case/coc.png)
+
+Once on the game, we can find him using his player id #88PPOOGG2. If we look at his clan details we find a link to a Discord server: [https://discord.gg/65CuhRczxj](https://discord.gg/65CuhRczxj)
+We need to prove we have joined the clan if we want to get some access role on the server, but we did not find anything interesting. However, by chatting with Raphael,
+and by pretending having the same passion for cycling, we managed to get a special role on the server, and we got access to specials channels.
+In the secret channel we found out he is second in
+command at the company. Also, the company received threatening emails
+for some time, and it kept getting worse. He forwarded email from Hortense:
+![Private message](/images/stranger_case/private.png)
+![Email](/images/stranger_case/threat.png)
+
+
+Thanks to this email we can now add extra information for the profile of [Hortense](#hl).
+Also, the sender of the mail use the following email: unknownuser89938487@proton.me
+
+
+
+#### Ulrich Jablonowski {#uj}
+
+{{< imgWithClass "/images/stranger_case/uj.png" "Ulrich Jablonowski" "uj.png" "" "stranger-case-pp" >}}
+
+- **Linkedin**: [https://www.linkedin.com/in/ulrich-jablonowski-0a17a5278/](https://www.linkedin.com/in/ulrich-jablonowski-0a17a5278/)
+
+Suspicious because he says the past few weeks have been good while all other received threats.
+
+#### Faye Terni {#ft}
+
+{{< imgWithClass "/images/stranger_case/ft.png" "Faye Terni" "ft.png" "" "stranger-case-pp" >}}
+
+- **Facebook**: [https://www.facebook.com/profile.php?id=100088501506551](https://www.facebook.com/profile.php?id=100088501506551)
+- **Linkedin**: [https://www.linkedin.com/in/faye-terni-9a7508259/](https://www.linkedin.com/in/faye-terni-9a7508259/)
+- **Email**: faye.terni@gmail.com
+
+Her older Facebook profile picture has a special instruction with hexiftool:
+FBMD0a000a700100004d1400002236000078390000853d00006449000085690000e96c0000cf7100002e7700000eae0000
```

Comment:
I think this was a useless/random artifact, right? Or was it actually relevant?
Also it's "exiftool", not "hexiftool".

Reply:
Yep can delete it there was no stegano in the chall.